install_config/topics/sssd_for_ldap_failover_configure_apache.adoc

```diff
@@ -0,0 +1,193 @@
+////
+configuring apache for sssd for ldap failover and extended attributes
+
+Module included in the following assemblies:
+
+* install_config/sssd_for_ldap_failover.adoc
+////
+
+[id='sssd-configuring-apache-{context}']
```

Comment:
Can we add a note that states that users should carefully read the Apache configuration files including all code comments and customize as needed.

Reply:
@enj, line 31.